library/azure_rm_trafficmanagerprofile.py

```diff
@@ -525,8 +424,7 @@ def ceate_update_traffic_manager_profile(self):
             profile_status=self.profile_status,
             traffic_routing_method=self.traffic_routing_method,
             dns_config=create_dns_config_instance(self.dns_config) if self.dns_config else None,
-            monitor_config=create_monitor_config_instance(self.monitor_config) if self.monitor_config else None,
-            endpoints=create_endpoints(self.endpoints)
```

Comment:
Not sure about this one. If no value is passed to endpoints when creating a Profile object, will updating the profile delete all endpoints?

Reply:
no, should pass what  get to the create_or_update